extension/chrome/elements/compose-modules/compose-pwd-or-pubkey-container-module.ts

```diff
@@ -58,7 +58,7 @@ export class ComposePwdOrPubkeyContainerModule extends ViewModule<ComposeView> {
     if (!this.view.recipientsModule.getRecipients().length || !this.view.sendBtnModule.popover.choices.encrypt) {
       this.hideMsgPwdUi(); // Hide 'Add Pasword' prompt if there are no recipients or message is not encrypted
       this.view.sendBtnModule.enableBtn();
-    } else if (this.view.recipientsModule.getRecipients().find(r => r.status === RecipientStatus.NO_PGP)) {
+    } else if (this.view.recipientsModule.getRecipients().find(r => [RecipientStatus.NO_PGP, RecipientStatus.REVOKED].includes(r.status))) {
```

Comment:
As revoked key makes encryption impossible (as opposed to expired key), we should affect sending button somehow, but the wording doesn't match with this change.
![image](https://user-images.githubusercontent.com/23335944/119143110-ea3d4a00-ba4f-11eb-8281-bc71c0f69014.png)
here it says "in grey":
![image](https://user-images.githubusercontent.com/23335944/119143195-ffb27400-ba4f-11eb-8c74-cd2179851eca.png)
Also, shouldn't the button be disabled if none of the recipients is PGP-capable?

Reply:
If none are pgp-capable, we still give the option to send password protected (the last sentence on the screenshot)

> As revoked key makes encryption impossible (as opposed to expired key), we should affect sending button somehow, but the wording doesn't match with this change.

Please change the wording as follows:

```
[emails (in grey) don't use encryption] [emails (in orange) have expired or revoked key]
Add their Public Key [or ask them to get FlowCrypt.]
Or send email password protected: _________
````

Where parts in square brackets are showing conditionally:
 - `emails (in grey) don't use encryption` -> show only if any NO_PGP present
 - `emails (in orange) have expired or revoked key` -> show only if any revoked keys present (I'm ignoring expired keys in the logic but still mentioning here because there may be mixed recipients)
 - `or ask them to get FlowCrypt.` -> show only if any NO_PGP present

Thank you for noticing this!

Reply:
Done.
Also, I noticed that class="no_pgp" has slightly different grey color (#989898) as opposed to "grey" class (#b7b7b7).
Should we make they same?

Reply:
Please choose the darker of the two. Thanks!